src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -691,6 +702,8 @@
     <None Include="App.config">
       <SubType>Designer</SubType>
     </None>
+    <EmbeddedResource Include="azurefunctions\proxy\SampleProxy.json" />
+    <EmbeddedResource Include="azurefunctions\proxy\RequestResponseOverride.json" />
```

Comment:
what are these json files? I ask because depending on if they need to be in the build output or not then I'll need to explicitly put them in the resulting zip

Reply:
never mind, I get it now.